fakestorage/server.go

```diff
@@ -155,6 +155,8 @@ func (s *Server) buildMuxer() {
 	r.Path("/b/{bucketName}").Methods("GET").HandlerFunc(s.getBucket)
 	r.Path("/b/{bucketName}/o").Methods("GET").HandlerFunc(s.listObjects)
 	r.Path("/b/{bucketName}/o").Methods("POST").HandlerFunc(s.insertObject)
+	r.Path("/b/{bucketName}/o/{objectName:.+}/acl").Methods("GET").HandlerFunc(s.listObjectAcl)
+	r.Path("/b/{bucketName}/o/{objectName:.+}/acl/{entity}").Methods("PUT").HandlerFunc(s.setObjectAcl)
```

Comment:
Why is entity in the request path? I thought it's a json object?

Reply:
it's the way google client sends it

Reply:
With the whole json object encoded in the url...?